packages/rum-core/src/domain/requestCollection.spec.ts

```diff
@@ -100,6 +100,33 @@ describe('collect fetch', () => {
     })
   })
 
+  describe('with feature flag fetch_duration', () => {
+    beforeEach(() => updateExperimentalFeatures(['fetch_duration']))
+    afterEach(() => resetExperimentalFeatures())
+
+    it('should still function if response is defined', (done) => {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏`still function` is quite vague, looking at other tests something like `should notify when response is defined` would be more consistent